src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.Storage;
+import duke.Ui;
+import exceptions.DukeException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    private Storage storage;
+    private Ui ui;
+
+    private Scanner sc = new Scanner(System.in);
+
+    public Duke() {
+
+        ui = new Ui();
+        this.storage = new Storage("./data/duke.txt");
+    }
+
+    public void startDuke() throws DukeException, IOException {
+        storage.loadData();
```

Comment:
I think these lines could be more readable. Maybe you could add empty lines between logical units?

Reply:
I agree, it would help to make the code more readable